scIB/integration.py

```diff
@@ -28,10 +28,12 @@ def runScanorama(adata, batch, hvg = None):
     import scanorama
     checkSanity(adata, batch, hvg)
     split = splitBatches(adata.copy(), batch)
-    emb, corrected = scanorama.correct_scanpy(split, return_dimred=True)
+    corrected = scanorama.correct_scanpy(split, return_dimred=True)
     corrected = corrected[0].concatenate(corrected[1:])
-    emb = np.concatenate(emb, axis=0)
-    corrected.obsm['X_emb']= emb
+    print(corrected.obs)
+    print(corrected.obsm)
+    print(corrected.var)
```

Comment:
could we remove the print statements here? @mumichae

Reply:
thanks, forgot about them